services/common/src/components/common/utils/helpers.ts

```diff
@@ -0,0 +1,602 @@
+import {
```

Comment:
This also exists in 

`services/common/src/redux/utils/helpers.ts` 💃🏻 🕺🏻

Reply:
Removed